rfcs/README.md

```diff
@@ -0,0 +1,116 @@
+first.
+
+## When you need to follow this process
+
+You never *need* to follow this process. In general, the tech lead is responsible for ensuring the
+application is well architected, tested, performant, and functional, subject to organizational
+constraints. However, this process is available to you if you have a good idea for how something
+should change. Such changes may include:
+
+* A substantial refactor, which takes a developer roughly an entire sprint or more.
+* Public facing changes, which might need to be coordinated with other groups inside or outside of
+  CTD.
+
+Changes that don't need an RFC may be:
+
+* Smaller changes, which can use the normal inbound Asana flow.
+
+When in doubt, feel free to ask!
+
+## Before creating an RFC
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality proposals, proposals for
+previously-rejected features, or those that don't fit into the near-term roadmap, may be quickly
+rejected, which can be demotivating for the unprepared contributor. Laying some groundwork ahead of
+the RFC can make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is generally a good idea to
+pursue feedback from other project developers beforehand, to ascertain that the RFC may be
+desirable; having a consistent impact on the project requires concerted effort toward
+consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking the idea over in our
+team Slack channel, or during your 1-on-1 with the tech lead. If the proposal is likely to require a
+chunk of time to write (for example, if you want to prove out and test a proof of concept or a part
+of it), it may be possible to allot sprint time to it. Submit an inbound Asana task and it will be
+reviewed and prioritized by the tech lead and product manager according to the road map and team
+priorities.
+
+## What the process is
+
+* Copy `rfcs/0000-template.md` to `rfcs/0000-my-feature.md` (where "my-feature" is descriptive).
+  Don't assign an RFC number yet; This is going to be the PR number and we'll rename the file
+  accordingly if the RFC is accepted.
+* Fill in the RFC. Put care into the details: RFCs that do not present convincing motivation,
+  demonstrate lack of understanding of the design's impact, or are disingenuous about the drawbacks
+  or alternatives tend to be poorly-received.
+* Submit a pull request. As a pull request the RFC will receive design feedback from the team, and
+  the author should be prepared to revise it in response.
+* Now that your RFC has an open pull request, use the issue number of the PR to update your 0000-
+  prefix to that number.
+* Assign the RFC to the tech lead, and announce it in the Slack channel for all the team to review.
+* RFCs rarely go through this process unchanged, especially as alternatives and drawbacks are shown.
```

Comment:
Maybe say here (or somewhere else) that it's worth getting feedback on an RFC even before proposing it?

Reply:
Added a final paragraph to the "Before creating an RFC" section with this.